app.js

```diff
@@ -31,4 +32,5 @@ app.use(function (err, req, res, next) {
   }
 })
 
-module.exports = app
+// module.exports = app
+export default app
```

Comment:
can we stick to either es5 or es6 way import/export?

Reply:
Yes sure.